etc/profile.d/codespace.sh

```diff
@@ -1,6 +1,22 @@
 # If not root
 if [ "$(whoami)" != "root" ]; then
 
+    # Check each environment variable and echo a warning message if it's not set
+    missing=0
+    env_vars=("CS50_TOKEN" "CS50_LANG" "CS50_TZ")
+    for var in "${env_vars[@]}"; do
+        if [[ -z "${!var}" ]]; then
+            echo -e "\e[31mWARNING: Missing environment variable $var\e[0m"
+            missing=1
+        fi
+    done
+
+    if [ $missing -eq 1 ]; then
+        echo "Please report this issue to sysadmins@cs50.harvard.edu with the following information:"
+        echo "GitHub username: $GITHUB_USER"
+        echo "Codespace name: $CODESPACE_NAME"
```

Comment:
Probably helpful to echo which vars are empty, else we'll just know that something is missing?